### DISCUSSION TITLE
搜索和索引不可用

### DISCUSSION THREAD

**Discussion by nxjonson** [AlistGo/alist#2599] [Q&A]
最新3.6版本的alist,点“搜索”和“索引”设置都提示：Search not available    怎么破？

**Comment by qianlima-666** [comment]
需要 设置 搜索索引

具体方法 
管理 --> 索引 --> 设置 --> 搜索索引 --> 选择 数据库 或 bleve --> 保存
当前索引 --> 重建索引

官方文档：https://alist.nn.ci/zh/guide/advanced/search.html

**Reply by nxjonson** [reply]
谢谢

**Reply by Eco1i** [reply]
感谢
